site/tsdb-engine/css/demo.css

```
@@ -164,7 +164,8 @@
 
 /* ── Dataset Panel ──────────────────────────────────────────────────── */
 
-.dataset-panel { }
+.dataset-panel {
+}
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)